torch/fx/operator_schemas.py

```diff
@@ -366,6 +366,16 @@ def normalize_function(
     if kwargs is None:
         kwargs = {}
     new_args_and_kwargs = None
+    if not isinstance(target, types.BuiltinFunctionType) and not (
+        isinstance(target, (OpOverloadPacket, OpOverload))
+    ) and hasattr(target, "_op"):
+        # The Executorch EXIR dialect targets have an underlying OpOverload which
+        # should be used for this function.
+        # Can't import EdgeOpOverload directly because of a circular dependency,
+        # so checking for "_op" existing is the next best thing.
```

Comment:
Can you edit this to say something like, "ExecuTorch's EdgeOpOverload are a wrapper around PyTorch's OpOverload, so we can unwrap it here to get its schema"

Reply:
Oh my bad I hadn't been monitoring the PR for a while, just the diff. Let me refresh this and address your comment

Reply:
Applied this comment to a new version